canvas_manage_course/migrations/0002_add_new_roles.py

```diff
@@ -10,8 +10,7 @@
 NEW_ROLES_MAP = {
     'Course Head': ['Head Instructor', 'Course Director'],
```

Comment:
Since 'Course Head' is renamed to 'Head Instructor' , we don't need a new set of permissions for the role. The role name should be renamed in the existing values. And if the rename happens first, make sure that the  'Course Director' refers to the correct role on line 11.
Also discussed this offline with Chris